### ISSUE TITLE
Prince's Tavern problem (maybe)

### ISSUE THREAD

**Issue by damienhurrell** [kdechant/eamon#26]
I cannot get into the vault in the Tavern. Is there something missing in the code, or am I missing something in how I'm playing?

**Comment by kdechant** [comment]
Did you find the numbers on the bathroom wall? The one labeled "V. Ault" is a red herring. There's another one that is similar to the name of the company that made the vault door.

Looking at the hints for that adventure, I think there's a hint missing that would have helped here. I'll have to review them more closely.

**Comment by damienhurrell** [comment]
I saw that but didn't make the connection! So I spent several hours searching elsewhere. I even searched the body in the shower four times, in case it was well hidden.

But I still can't open the damn door. I've used every combination of open, use, examine, put and say with the words vault, door, lock, combination and ZVC I can think of without success. I note that the how to play instructions include the command "unlock", but the game doesn't recognise that command. 

I agree that a hint would be useful - a multilayered one for preference. Something like "The prince has a very bad memory and has written the combination somewhere he visits often" followed by "The prince is cunning and has obscured the combination amongst some red herrings. The correct combination is identified by a name that is similar to the manufacturer of the door" followed by explicit instructions.

**Comment by kdechant** [comment]
I'm going to give the Prince's Tavern a thorough test soon. I'll see if I can figure out what's going on, and update the hints to be more useful.

**Comment by kdechant** [comment]
Well, this is embarrassing. It turns out that the vault door somehow got categorized as a "treasure" instead of as a "door" so it couldn't be opened. I fixed that, so it will now prompt for the combination when you type "OPEN VAULT DOOR".

I also added more information to the hints for the adventure and made it possible to blur out hints that are more spoiler-y, until the player clicks on the "show" link to see the text.
